hw/drivers/sensors/dps368/syscfg.yml

```diff
@@ -0,0 +1,61 @@
+#Copyright 2019 Infineon Technologies AG
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+#
+
+syscfg.defs:
+    DPS368_CLI:
+        description: 'Enable shell support for the DPS368'
+        value: 1
```

Comment:
CLI's are typically disabled by default, and are enabled by overriding the setting in the user's target.

Reply:
During our test we left it enabled. This has been fixed.